docs/developing-contracts/examples/feeds.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Gear Feeds Workshop Contract'
```

Comment:
I suggest to describe as a contract example among others, to demonstrate Gear's capabilities. We might conduct different workshops with different contracts, not only this one.
Also don't you think the "Feed" word can be misinterpreted? Maybe it is better to use something like - Messenger?